docs/tutorial/part03.rst

```diff
@@ -2,5 +2,295 @@
+
+    .. _django-neomodel: https://github.com/neo4j-contrib/django-neomodel
+
+    ``class Entity(StructuredNode):``
+
+
+3. The next step is to add the properties for the node. Each node property in the Neo4j database
+should be a property in the model class.
+
+The properties of the ``Entity`` node in the database have the following scheme::
+
+        {
+          "sourceID": "Panama Papers",
+          "address": "MEI SERVICES LIMITED ROOM E; 6TH FLOOR; EASTERN COMMERCIAL CENTRE; 395-399 HENNESSY ROAD HONG KONG",
+          "jurisdiction": "SAM",
+          "service_provider": "Mossack Fonseca",
+          "countries": "Hong Kong",
+          "jurisdiction_description": "Samoa",
+          "valid_until": "The Panama Papers data is current through 2015",
+          "ibcRUC": "25475",
+          "name": "JIE LUN INVESTMENT LIMITED",
+          "country_codes": "HKG",
+          "incorporation_date": "10-APR-2006",
+          "node_id": "10000020",
+          "status": "Active,
+        }
+
+Therefore, the structure of the ``Entity`` class should be::
+
+    class Entity(StructuredNode):
+        sourceID                           = StringProperty()
+        address                            = StringProperty()
+        jurisdiction                       = StringProperty()
+        service_provider                   = StringProperty()
+        countries                          = StringProperty()
+        jurisdiction_description           = StringProperty()
+        valid_until                        = StringProperty()
+        ibcRUC                             = StringProperty()
+        name                               = StringProperty()
+        country_codes                      = StringProperty()
+        incorporation_date                 = StringProperty()
+        node_id                            = StringProperty(index = True)
+        status                             = StringProperty()
+
+On our case, the database we are using only have string values for the properties. However, there
+are several types of properties available; such as, IntegerProperty, ArrayProperty, DateProperty.
+In case you need to know more about this, go to this link_
+
+    .. _link: http://neomodel.readthedocs.io/en/latest/properties.html
+
+
+4. Add the relationships for the node::
+
+    officers                 = RelationshipFrom('.officer.Officer', 'OFFICER_OF')
+    intermediaries           = RelationshipFrom('.intermediary.Intermediary', 'INTERMEDIARY_OF')
+    addresses                = RelationshipTo('.address.Address', 'REGISTERED_ADDRESS')
+    others                   = RelationshipFrom('.other.Other', 'CONNECTED_TO')
+
+.. [*]  You can find all the nodes relation by opening your local database, click HERE_ to access it. When you get indise, run the command ``CALL db.schema()`` on the search bar to display the relationships.  note: you need to have it up and running to work. 

```

Comment:
Here; do you mean `inside` instead of `indise`? 🤔

Reply:
yes sir